sopel_modules/troet/troet.py

```diff
@@ -4,14 +4,96 @@
 from sopel.bot import Sopel, SopelWrapper
 from sopel.tools import get_logger
 from sopel.trigger import Trigger
-from sopel_modules.mastodon.structures import LimitedSizeDict, strip_tags
+
+from collections import OrderedDict
+from io import StringIO
+from html.parser import HTMLParser
+
+from sopel.tools import get_logger
+from sopel.db import SopelDB
 
 from mastodon import Mastodon, MastodonNotFoundError
 
 PLUGIN_OUTPUT_PREFIX = "[troooet] "
 
 LOGGER = get_logger(__name__)
 
+class LimitedSizeDict(OrderedDict):
```

Comment:
I don't like this change. It's already a quite chaotic 300 line file and by adding 100 lines for helper functions this is not notably improved. But this is just a small tool, I'm not a developer and I don't feel like changing this now, so I'll approve it.